src/components/battles/animations/shared/effects/Damage.ts

```diff
@@ -27,7 +27,9 @@ export class Damage<TGameStartr extends FullScreenPokemon> extends Component<TGa
     public run(teamAndAction: ITeamAndAction<IMoveAction>, effect: IDamageEffect, onComplete: () => void): void {
         const statistic: IStatistic = teamAndAction.target.actor.statistics.health;
         const amount: number = this.calculator.calculateDamage(teamAndAction, effect);
+        const attackingPokemon: IActor = teamAndAction.source.actor;
 
+        this.updateBattleParticipants(attackingPokemon, teamAndAction);
```

Comment:
A Pokemon actually gets added to the list of participants if it sees the battle at all, not just if it makes a move.